CHANGELOG.md

```diff
@@ -14,6 +14,7 @@
 
 - Fixed the T2A CPU template not to unset the MMX bit (CPUID.80000001h:EDX[23])
   and the FXSR bit (CPUID.80000001h:EDX[24]).
+- Fixed the T2A CPU template to set the MMX bit (CPUID.80000008h:EBX[2]).
```

Comment:
nit: s/MMX/RstrFpErrPtrs/

Reply:
Good catch! Looks like I've been caught red-handed copying and pasting the previous line, right? 😆

Reply:
Who knows, maybe you just run a script that creates all these patches? ;)